### PR TITLE
Decisiontree updated to use live backend

### PR DESCRIPTION
**Pull request by tkenakka**
Working version.
Error handling etc. may need further adjustments.
Votings not yet implemented.

### CODE REVIEW

**Comment by tkenakka** [comment]
Notice: remember to take an update for the config files

File: app/scripts/decisions.directive.js [hunk 1]
```diff
@@ -23,42 +23,35 @@ angular.module('dashboard')
...
-                        $log.error("dbDecisions.getDecisions");
+                    AhjoMeetingSrv.getDecisions(aMtg.meetingGuid, aTopic.topicGuid).then(function (resp) {
+                        $log.log("dbDecisions.getDecisions", resp);
+
+                        self.record = resp.record;
+                        self.supporter = resp.supporter;
+                        self.voting = resp.voting;
+                    }, function (error) {
+                        // $log.error("dbDecisions.getDecisions");
+                        Utils.processAhjoResponse(error);
```
Comment (tkenakka) [inline app/scripts/decisions.directive.js]: This needs to be updated in further PRs to handle the custom error object now passed by the ui service.

**Comment by evidentti** [comment]
Utils.processAhjoResponse is not a function. (In 'Utils.processAhjoResponse(error)', 'Utils.processAhjoResponse' is undefined)

File: app/scripts/services/ahjo.meeting.srv.js [hunk 2]
```diff
@@ -12,10 +12,14 @@
 * Service in the dashboard.
 */
 angular.module('dashboard')
-    .service('AhjoMeetingSrv', ['$log', '$http', 'ENV', '$q', '$timeout', function ($log, $http, ENV, $q, $timeout) {
+    .service('AhjoMeetingSrv', ['$log', '$http', 'ENV', '$q', '$timeout', 'Utils', 'CONST', function ($log, $http, ENV, $q, $timeout, Utils, CONST) {
         $log.log("AhjoMeetingSrv: SERVICE");
         var self = this;
 
+        function rejectError(aD, aCode) {
```
Comment (evidentti) [inline app/scripts/services/ahjo.meeting.srv.js]: Is this needed?

File: app/scripts/services/ahjo.meeting.srv.js [hunk 3]
```diff
@@ -59,193 +63,46 @@ angular.module('dashboard')
...
+                    method: 'GET',
+                    cache: false,
+                    url: ENV.AhjoApi_Decisions.replace(':meetingguid', aMeetingGuid).replace(':topicguid', aTopicGuid)
+                }).then(function (resp) {
+
+                    var errCode = Utils.processAhjoError(resp, false);
+                    if (errCode) {
+                        $log.error("AhjoMeetingSrv.getDecisions: REST error", errCode);
+                        rejectError(errCode);
+                    }
```
Comment (evidentti) [inline app/scripts/services/ahjo.meeting.srv.js]: It seems that execution continues to 'objects' part even 'errorCode' is found. Maybe 'reject' works as 'return'? Or is there need for 'else'?

**Comment by evidentti** [comment]
LGTM

**Pull request merged**